src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,21 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engine;
+    private String color;
+    private int maxSpeed;
+    private int doors;
+    private boolean isNew;
 
+    public Car(Engine engine, String color, int maxSpeed, int doors, boolean isNew) {
+        this.isNew = isNew;
+        this.doors = doors;
+        this.color = color;
+        this.maxSpeed = maxSpeed;
+        this.isNew = isNew;
+    }
+
+    public Car clone() {
+        return new Car(engine, color, maxSpeed, doors, isNew);
```

Comment:
you will get the new Car object, but what about it's engine?

Reply:
it's by mistake

Reply:
dome